### PR TITLE
Frontend updates

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**
 * Add resource selector
 * Virtual delivery method field added
 * Fix styling (margins colors etc)
 * Copy updates

**How to test**
1) Pull down
2) Compare the frontend against [the design](https://preview.uxpin.com/ad995fd722bd26c0f9a1c8a115f1b41833637308#/pages/134880468/simulate/sitemap)

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/283
* https://github.com/HHS/Head-Start-TTADP/issues/308
* https://github.com/HHS/Head-Start-TTADP/issues/330

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

**Review by dcmcand — COMMENTED** [review]
Since we now only have one type of file upload, should we change the file model to get rid of the attachment type field?

File: frontend/src/App.css [hunk 1]
```diff
@@ -30,6 +30,10 @@ body {
   background-color: #f8f8f8;
 }
 
+.text-ink {
+  color: #21272D
```
Comment (dcloud) [inline frontend/src/App.css:34]: IDK if y'all have discussed this, but if you converted your CSS to Sass, you could [override the USWDS theme token colors](https://designsystem.digital.gov/design-tokens/color/theme-tokens/#customizing-theme-color-tokens) (and do a lot of other things that could help manage styles).

File: frontend/src/pages/ActivityReport/index.css [hunk 2]
```diff
@@ -1,13 +1,29 @@
-.smart-hub--form-section {
-  margin-top: 2rem;
+.smart-hub--form-group {
+  margin-top: 1.5rem;
```
Comment (dcloud) [inline frontend/src/pages/ActivityReport/index.css:2]: I think that's equal to [`.margin-105`](https://designsystem.digital.gov/utilities/margin-and-padding/) in USWDS-speak.

File: frontend/src/pages/ActivityReport/index.css [hunk 3]
```diff
@@ -1,13 +1,29 @@
...
 }
 
 .smart-hub--report-checkbox {
   margin-top: 1.25rem;
 }
 
+.smart-hub--form-section {
+  margin-top: 2rem;
```
Comment (dcloud) [inline frontend/src/pages/ActivityReport/index.css:10]: [`.margin-top-4`](https://designsystem.digital.gov/utilities/margin-and-padding/)

File: frontend/src/pages/ActivityReport/index.css [hunk 4]
```diff
@@ -1,13 +1,29 @@
...
+.smart-hub--virtual-delivery-group {
+  border-left-color: #0166ab;
+  border-left-style: solid;
+  border-left-width: 4px;
+  padding-left: 10px;
+}
+
 .smart-hub--report-legend .usa-legend {
-  font-family: Merriweather;
+  font-family: 'Source Sans Pro Web';
```
Comment (dcloud) [inline frontend/src/pages/ActivityReport/index.css:26]: I noticed that `.smart-hub--form-required` has this as `SourceSansPro` which doesn't work in Firefox, so the *Required* labels are rendering as Times.

Also, there are [`.font-ui` and `.font-family-sans` classes](https://designsystem.digital.gov/utilities/font-size-and-family/) that achieve this.

**Review by dcloud — COMMENTED** [review]
The new resources fields look neat! I loaded this up before realizing I needed to run a db migration to get those changes, but overall this looks great. 

The trashcan icon on the Goals on the Goals page are light grey. Should those look more like these darker trashcans on the Topics page?

It would be great if you could fix the font-family issue on the `.smart-hub--form-required` elements (they render with TimesNewRoman in Firefox). I think that `font-family: SourceSansPro;` should be unnecessary anyway?

Otherwise my other comments are around trying to influence everyone to use the classes that the USWDS provides rather than defining our own. So take those as what they are: suggestions

File: src/migrations/20210302170502-add-resources-to-activity-report.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn('ActivityReports',
```
Comment (gopar) [inline src/migrations/20210302170502-add-resources-to-activity-report.js:4]: Does it make any sense to wrap these in a `Promise.all(...)`? I remember a comment about this from a while ago

**Review by gopar — APPROVED** [review]

**Review by dcmcand — APPROVED** [review]
:rocket:

File: .circleci/config.yml [hunk 6]
```diff
@@ -339,10 +339,10 @@ jobs:
                 hses_data_username: HSES_DATA_USERNAME
                 hses_data_password: HSES_DATA_PASSWORD
             - run:
-                name: Undo database seeding, then migrate and seed
+                name: Migrate database
                 command: |
                   cf run-task tta-smarthub-dev \
-                    --command "yarn db:migrate:prod && yarn db:seed:undo:prod && yarn db:seed:prod" \
+                    --command "yarn db:migrate:prod" \
```
[the remark this answers is not in the document]
Comment (rahearn) [reply .circleci/config.yml:345]: Let's go ahead and do that. I think this covers things well enough for the next couple weeks.

**Review by rahearn — APPROVED** [review]
🎉

**Pull request merged**